docs/graphcast/radios/subgraph-radio/subgraph_upgrade.md

```diff
@@ -0,0 +1,34 @@
+---
+sidebar_position: 3
+---
+
+# Subgraph upgrade pre-sync feature
```

Comment:
include a workflow diagram here; you can edit from the existing one https://github.com/graphops/subgraph-radio/issues/6